src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/services/NodeService.kt

```diff
@@ -116,7 +117,9 @@ class NodeService(
                                   DateTimeFormatter.ISO_OFFSET_DATE_TIME)
                                 .truncatedTo(ChronoUnit.SECONDS)
                                 .format(DateTimeFormatter.ISO_LOCAL_DATE_TIME)
-                            this.rrn = authCompleted.transactionAuthorizationCompletedData.rrn
+                            this.rrn = getRrnForClosePaymentByAuthorization(authCompleted.transactionAuthorizationRequestData.paymentGateway,
```

Comment:
Have we to trim also the rrn passed into TransactionDetailsDto, right? (line 145)